server/chat-commands/info.ts

```diff
@@ -62,14 +64,15 @@ export const commands: ChatCommands = {
 			if (roomid === 'global') continue;
 			const targetRoom = Rooms.get(roomid)!;
 
-			const authSymbol = (targetRoom.auth && targetRoom.auth[targetUser.id] ? targetRoom.auth[targetUser.id] : '');
+			const authSymbol = (targetRoom.settings.auth && targetRoom.settings.auth[targetUser.id] ?
+				targetRoom.settings.auth[targetUser.id] : '');
```

Comment:
```suggestion
			const authSymbol = targetRoom.settings.auth[targetUser.id] || '';
```

Reply:
Making `auth` guaranteed to exist actually simplifies a lot of code along these lines.